storybook/src/data/documents.data.js

```diff
@@ -0,0 +1,66 @@
+import uuid from 'uuid/v4';
+
+export default class Document {
+  constructor(build) {
```

Comment:
We are still thinking in JSON, which is completely valid! But, when using classes, we would typically store something like `this.entityType = 'document' ` and also have a _getter_.
Anyway, we just need to align on what we pretend.